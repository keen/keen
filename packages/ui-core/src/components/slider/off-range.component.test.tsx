import React from 'react';
import { shallow } from 'enzyme';
import OffRange from './off-range.component';

describe('<OffRange />', () => {
  const props = {
    left: 0,
    size: 100,
  };

  it('should render correctly horizontal component', () => {
    const wrapper = shallow(<OffRange isHorizontal={true} {...props} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly vertical component', () => {
    const wrapper = shallow(<OffRange isHorizontal={false} {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
