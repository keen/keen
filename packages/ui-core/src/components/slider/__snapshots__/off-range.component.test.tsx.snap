// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OffRange /> should render correctly horizontal component 1`] = `
<ForwardRef(MotionComponent)
  style={
    Object {
      "background": "#E1E2E4",
      "borderRadius": 3,
      "height": undefined,
      "left": 0,
      "position": "absolute",
      "top": 0,
      "width": 100,
    }
  }
/>
`;

exports[`<OffRange /> should render correctly vertical component 1`] = `
<ForwardRef(MotionComponent)
  style={
    Object {
      "background": "#E1E2E4",
      "borderRadius": 3,
      "height": 100,
      "left": 0,
      "position": "absolute",
      "top": 0,
      "width": undefined,
    }
  }
/>
`;
